src/utils/utils.ts

```diff
@@ -40,95 +41,169 @@ export function diff(a: any[], b: any[]) {
+  return new Promise(async (resolve, reject) => {
+    createApp(
+      folderPath,
+      'https://github.com/sasjs/react-seed-app',
+      'https://github.com/sasjs/docs',
+      (err: string) => {
+        return reject(new Error(err))
+      }
+    )
 
     return resolve()
   })
 }
 
 export async function createAngularApp(folderPath: string): Promise<void> {
-  return new Promise(async (resolve, _) => {
-    createApp(folderPath, 'https://github.com/sasjs/angular-seed-app.git')
+  return new Promise(async (resolve, reject) => {
+    createApp(
+      folderPath,
+      'https://github.com/sasjs/angular-seed-app',
+      'https://github.com/sasjs/docs',
+      (err: string) => {
+        return reject(new Error(err))
+      }
+    )
     return resolve()
   })
 }
 
 export async function createMinimalApp(folderPath: string): Promise<void> {
-  return new Promise(async (resolve, _) => {
-    createApp(folderPath, 'https://github.com/sasjs/minimal-seed-app.git')
+  return new Promise(async (resolve, reject) => {
+    createApp(
+      folderPath,
+      'https://github.com/sasjs/minimal-seed-app',
+      'https://github.com/sasjs/docs',
+      (err: string) => {
+        return reject(new Error(err))
+      }
+    )
     return resolve()
   })
 }
 
 export async function createTemplateApp(folderPath: string, template: string) {
   return new Promise<void>(async (resolve, reject) => {
-    const { stdout, stderr, code } = shelljs.exec(
-      `git ls-remote https://username:password@github.com/sasjs/template_${template}.git`,
-      { silent: true }
+    const { stdout, stderr, code } = downloadFile(
+      `https://username:password@github.com/sasjs/template_${template}`
     )
 
-    if (stderr.includes('Repository not found') || code) {
+    if (stderr.includes('404: Not Found') || code) {
       return reject(new Error(`Template "${template}" is not a SASjs template`))
     }
 
-    if (!stdout) {
-      return reject(new Error(`Unable to fetch template "${template}"`))
-    }
-
-    createApp(folderPath, `https://github.com/sasjs/template_${template}.git`)
+    createApp(
+      folderPath,
+      `https://github.com/sasjs/template_${template}`,
+      'https://github.com/sasjs/docs',
+      (err: string) => {
+        return reject(new Error(err))
+      }
+    )
     return resolve()
   })
 }
 
+/**
+ * This function will first download / unzip the target app from repoUrl, then download / unzip the sasjs/docs repo into a subdirectory named 'docs'.
+ * Finally, if installDependencies is true, the npm install command will be executed.
+ * @param folderPath full path of the directory where desired seed app will be created
+ * @param repoUrl sasjs/{seed-app} repo url
+ * @param docsUrl sasjs/docs repo url
+ * @params installDependencies whether or not to do `npm install`
+ */
 function createApp(
   folderPath: string,
   repoUrl: string,
+  docsUrl: string,
+  errorCallback: (err: string) => void,
```

Comment:
`errorCallback` is the same for all app, so `createApp` can just reject promise with error by default